File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height = 0;
    do
    {
        height = get_int("Height:");
    }
    while (height < 1 || height > 8);

    int width = (height * 2) + 2;

    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < height; j++)
        {
            if (j < (height - (i + 1)))
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }

        printf("  ");

        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int


def piramid(quantity, base):
    if quantity > 0:
        piramid(quantity - 1, base)
        print(" " * (base - quantity) + "#" * quantity, end="")
        print("  ", end="")
        print("#" * quantity)


while True:
    quantity = get_int("Height: ")
    if quantity > 0 and quantity < 9:
        break


piramid(quantity, quantity)